pkg/controller/route/route_controller.go

```diff
@@ -88,11 +89,42 @@ func New(routes cloudprovider.Routes, kubeClient clientset.Interface, nodeInform
 		clusterName:      clusterName,
 		clusterCIDR:      clusterCIDR,
 		nodeLister:       nodeInformer.Lister(),
+		nodeInformer:     nodeInformer.Informer(),
 		nodeListerSynced: nodeInformer.Informer().HasSynced,
```

Comment:
Seems like we can get rid of `nodeListerSynced` since we can reference HasSynced from `nodeInformer`